internal/model/matcher.go

```diff
@@ -73,6 +73,9 @@ func NewCompositeMatcher(matchers []PathMatcher) PathMatcher {
 
 func (c CompositePathMatcher) Matches(f string, isDir bool) (bool, error) {
 	for _, t := range c.Matchers {
+		if t == nil {
```

Comment:
this seems wrong. we should disallow nils in the matcher repr

Reply:
Yeah, I need to track down where this is coming from.